server/db/__init__.py

```diff
@@ -57,7 +57,8 @@ async def deadlock_retry_execute(conn, *args, max_attempts=3):
         try:
             return await conn.execute(*args)
         except OperationalError as e:
-            if any(msg in e.message for msg in (
+            error_text = str(e)
+            if any(msg in error_text for msg in (
```

Comment:
I'm curious, what's more advantageous about using `str(e)` over referencing the error message directly?

Reply:
It doesn't throw an attribute error xD. I think `str(e)` is the only standard way to get the error message from an exception, exceptions don't have a `message` attribute.